src/libraries/System.Collections/System.Collections.sln

```diff
@@ -194,28 +204,25 @@ Global
 		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Release|x64.Build.0 = Release|Any CPU
 		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Release|x86.ActiveCfg = Release|Any CPU
 		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Release|x86.Build.0 = Release|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|Any CPU.ActiveCfg = Debug|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|Any CPU.Build.0 = Debug|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|x64.ActiveCfg = Debug|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|x64.Build.0 = Debug|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|x86.ActiveCfg = Debug|Any CPU
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA}.Checked|x86.Build.0 = Debug|Any CPU
 	EndGlobalSection
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(NestedProjects) = preSolution
 		{4385739F-14A1-4F21-96E3-12EA90580575} = {C848C534-DD16-4A69-81FF-894456E4B099}
-		{DF501250-4105-4F79-A508-90B3A13A975A} = {C848C534-DD16-4A69-81FF-894456E4B099}
 		{38D8F258-655E-459B-B2CE-0DED144AFBF7} = {6CD2C5E3-B5BA-4219-B8EF-9B5D486BF805}
-		{BB54ED9D-FF71-4D91-B7C0-984AB0976798} = {6CD2C5E3-B5BA-4219-B8EF-9B5D486BF805}
 		{77E9C47D-4AC4-4402-8613-40DF427174BD} = {06AE90FB-F007-4E70-BCD2-5CAD106A7C97}
-		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA} = {06AE90FB-F007-4E70-BCD2-5CAD106A7C97}
+		{DF501250-4105-4F79-A508-90B3A13A975A} = {C848C534-DD16-4A69-81FF-894456E4B099}
+		{BB54ED9D-FF71-4D91-B7C0-984AB0976798} = {6CD2C5E3-B5BA-4219-B8EF-9B5D486BF805}
 		{F2E57833-4968-430D-8149-733DE03A7314} = {B958B834-96D2-4B18-AA42-22B312BE9925}
 		{389B3CD7-FF7C-4004-A6BA-29B39B4BDF3A} = {B958B834-96D2-4B18-AA42-22B312BE9925}
 		{00398A78-1620-4996-9922-C82D1AF84714} = {B958B834-96D2-4B18-AA42-22B312BE9925}
+		{EE21C87C-5515-45E1-9F94-F11BE4CEE2CA} = {06AE90FB-F007-4E70-BCD2-5CAD106A7C97}
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {FDD40D53-AF3E-4A20-BE17-7555EB5494F9}
 	EndGlobalSection
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		..\System.Private.CoreLib\src\System.Private.CoreLib.Shared.projitems*{4385739f-14a1-4f21-96e3-12ea90580575}*SharedItemsImports = 5
```

Comment:
Are we ok with these changes?

Reply:
Auto updated by VS. If CI is happy, I'm fine with it. It's targeting `main` anyway so we can always change if it creates problems in the future.